spec/user_spec.cr

```diff
@@ -22,6 +22,23 @@ module PlaceOS::Model
       end
     end
 
+    describe "#destroy" do
+      it "raises if only one User remains" do
+        User.clear
+        user = Generator.user.save!
+        expect_raises(Model::Error, "At least one user must remain") do
+          user.destroy
+        end
+      end
+
+      it "does not raise if more than one User remains" do
+        User.clear
+        user0 = Generator.user.save!
+        Generator.user.save!
```

Comment:
duplicate?

Reply:
@dukeraphaelng this is generatating two user's, then deleting one.